### PR TITLE
add dbver & dbver.py

### PR DESCRIPTION
**Pull request by jasonjoo2010**

### CODE REVIEW

**Comment by jasonjoo2010** [comment]
the whole database will be created by version 1.
so if there are existing data it will fail.
you can manually test and insert the correct version pointer into [dbver] if there is no difference between existing structure and version 1.
eg: 
`insert into dbver (repository_id, repository_path, version) values('redisctl', '/home/hblzxsj/svn/redis-ctl/dbver', 1);`

**Comment by zheplusplus** [comment]
Cool. Thanks for your effort.

Could you please squash those 2 commits into one, and check it with PEP8?

File: dbver.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        meta = sqlalchemy.MetaData()
+        engine = get_engine()
+        meta.bind = engine
+        tables = meta.tables
+        if len(tables) == 0:
+            db_version_control(000)
+            return versioning_api.db_version(get_engine(), repository)
+        else:
+            # db has tables without migrate_version, it out of control
+            raise
```
Comment (zheplusplus) [inline dbver.py:44]: According to [PEP8](https://www.python.org/dev/peps/pep-0008/#blank-lines) there should be blank lines after top level function definitions.

**Comment by jasonjoo2010** [comment]
all have been done.

next I will add support for other proxy, how would you like supported by new type specification during proxy adding or just auto-detect?

**Comment by zheplusplus** [comment]
I think it will be easier to understand by adding a program type column in the `proxy` table.

**Comment by jasonjoo2010** [comment]
fine.
I will first try to add supporting for corvus.

**Pull request merged**